styles.css

```diff
@@ -114,3 +133,356 @@ p {
   gap: 20px;
   margin-top: 30px;
 }
+
+/* ----------------mobile version---------------- */
+
+/* my recent works */
+
+.main-1st-Title {
+  display: grid;
+  align-items: center;
+  width: 100%;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 40px 24px;
+  padding-left: 25px;
+  color: black;
+}
+
+.underlined {
+  margin: 30px;
+}
+
+.recent-pic {
+  align-items: center;
+  padding-left: 30px;
+  width: 100%;
+}
+
+.top-stories {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  margin: 0 0 24px 24px;
+  margin-bottom: 12px;
+  width: 327px;
+  height: 36px;
+  color: #091e42;
+}
+
+.main-section {
```

Comment:
- [ ] Please add a CSS grid system to the `main section` as suggested by the previous reviewer. 😄

Reply:
resolved